app/helpers/fhir_helper.rb

```diff
@@ -134,6 +146,9 @@ def patient_from_fhir(patient, default_jurisdiction_id)
       american_indian_or_alaska_native: race_code?(patient, '1002-5'),
       asian: race_code?(patient, '2028-9'),
       native_hawaiian_or_other_pacific_islander: race_code?(patient, '2076-8'),
+      race_other: race_code?(patient, 'OTH'),
+      race_unknown: race_code?(patient, 'UNK'),
+      race_refused_to_answer: race_code?(patient, 'ASKU'),
```

Comment:
This needs to be updated per the corresponding changes you made in the `to_us_core_race` method. I believe this explains the issue Kylie noted [here](https://github.com/SaraAlert/SaraAlert/pull/689#issuecomment-783705532).

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/689/commits/767b8765f1830cb0e34e8bc8b1adbbe48c0786f5#diff-402d451d1f7019322100522f6898c3153444edb8ddb963e181a457c84a2ca6b6R152